### PR TITLE
test(Browser): Addl std features category

### PR DESCRIPTION
**Pull request by barbnewrelic**
A test to get rid of the "Additional standard features" taxonomy. No changes to the Japanese/Korean taxonomy in the first build, so I'm not sure if this will work or not.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by barbnewrelic** [comment]
@zuluecho9  I thought I updated the JP and KR folders correctly but the build failed again. When you have some free time, can you check? No rush. Thanks!

**Review by zuluecho9 — DISMISSED** [review]

File: src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx [hunk 1]
```diff
@@ -11,28 +18,38 @@ import BrowserGeoZoom from 'images/browser-geo-zoom-051822.png'
...
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Filterable geos:** Get detailed information about network performance and end users' experience anywhere around the world.
+</figcaption>
+
 To view or sort the performance information by location:
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Filterable geography**.
+1. Go to **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Filterable geos**.
 2. Use the dropdown to select the type of performance or usage data you want to view, such as page load or unique session count, average time for various processes, largest contentful paint, first input delay, cumulative layout shift, etc.
-3. To drill down to a [specific area](#functions), mouse over or select any area on the geographical map, or select any of the ten worst performing locations from the list to the right of the map.
```
Comment (jodinkansagor) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx:35]: I believe this table is all of the reporting countries, so not just a top ten. Also, it is not always the worst performing. It is only the worst performing when high numbers equals bad. For page load count and unique session count, the top of the list is the best performing.

Comment (barbnewrelic) [reply src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx:35]: @jodinkansagor  Yikes, I will make that correction. Thanks for setting me straight!

**Comment by barbnewrelic** [comment]
I need this taxonomy on the docs site so i can make some other edits, so I'll merge this branch now.

**Pull request merged**